### ISSUE TITLE
Generate & use GUIDs for monsters

### ISSUE THREAD

**Issue by PouncySilverkitten** [PouncySilverkitten/kobold-training-club#17]

**Comment by vincentscode** [comment]
I agree that using IDs instead of names for monsters would really improve the frontend architecture.
Every monster already has a FID though, so generation is not necessary I think.

**Comment by struandw** [comment]
Ooh, now I’m curious; how do you think front end architecture could be improved with GUIDs or FIDs? I originally needed them to solve a backend issue, which I found another way around. 

My concern about using FIDs is that they’re generated by ‘users’ which offers a fun new route for errors - if I write code to generate them I can be certain that there will only ever be one FID per monster, and only one monster per FID.

**Comment by struandw** [comment]
I appreciate the suggestions - I’m working on Random Encounter Generation just now, and after that mobile optimisation, but if I ever get around to streamlining the front end code I’ll definitely bear this in mind.
